src/test/java/pages/HowItWorksPage.java

```diff
@@ -0,0 +1,12 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class HowItWorksPage extends BasePage {
+    public HowItWorksPage(WebDriver driver) {
+        super(driver);
+    }
+
+    public By pageTitle = By.cssSelector(".navbar-header a.navbar-brand");
```

Comment:
All class variables must be before constructor. This variable should private or protected (if you want to use it in the descendant class) and final.